commit cf323c2e4708a9ca86a62216505f82f9a0d4f8fe
Author: Camilo Higuita <chiguitar@unal.edu.co>
Date:   Wed Nov 30 11:02:52 2022 -0500

    organize appstream releases tag in desc order. add property to toggle auto resuming last playlist

diff --git a/org.kde.vvave.appdata.xml b/org.kde.vvave.appdata.xml
index 9d3b69c..cd9569b 100644
--- a/org.kde.vvave.appdata.xml
+++ b/org.kde.vvave.appdata.xml
@@ -167,18 +167,18 @@
   </recommends>
   <project_group>KDE</project_group>
   <releases>
-    <release version="1.2.2" date="2021-05-11">
-      <url>https://nxos.org/maui/maui-1-2-2-release/</url>
-    </release>
-    <release version="0.1.0" date="2018-09-06"/>
-    <release version="2.1.2" date="2022-05-16">
-      <url>https://mauikit.org/blog/maui-2-1-2-release/</url>
+ <release version="2.2.1" date="2022-11-25">
+      <url>https://mauikit.org/blog/maui-2-2-1-release/</url>
     </release>
-    <release version="2.2.0" date="2022-08-30">
+<release version="2.2.0" date="2022-08-30">
       <url>https://mauikit.org/blog/maui-2-2-0-release/</url>
     </release>
-    <release version="2.2.1" date="2022-11-25">
-      <url>https://mauikit.org/blog/maui-2-2-1-release/</url>
+<release version="2.1.2" date="2022-05-16">
+      <url>https://mauikit.org/blog/maui-2-1-2-release/</url>
+    </release>
+    <release version="1.2.2" date="2021-05-11">
+      <url>https://nxos.org/maui/maui-1-2-2-release/</url>
     </release>
-  </releases>
+    <release version="0.1.0" date="2018-09-06"/>
+ </releases>
 </component>
